llama.cpp/ggml-quants.inc

```diff
@@ -1950,6 +1950,44 @@ void quantize_row_q2_K_ref(const float * restrict x, block_q2_K * restrict y, in
 
     const float q4scale = 15.f;
 
+    // [kawrakow] Detect TriNet
```

Comment:
Could you add a an `// [kawrakow] ...` comment to every line that's been changed under `llama.cpp/`? That makes it easy for me to synchronize sources. Thanks!

Reply:
Should be good now.